include/mfmg/common/hierarchy.hpp

```diff
@@ -181,6 +183,13 @@ class Hierarchy
 
       if (level_index == num_levels - 1)
       {
+        if (level_index == 0)
+        {
+          // When using ML for the full hierarchy, do not zero out initial guess
+          params->put("coarse.params.zero starting solution",
```

Comment:
@aprokop I could not find `"zero starting solution"` in ML's documentation.  Where did you find out about it?

Reply:
I could not find anything about using ML in standalone mode in the documentation either. I had to dig in in the source code of `ml/src/Utils/ml_MultiLevelPreconditioner.{h,cpp}` to find out about it.